app/assets/javascripts/components/overview/details.cjsx

```diff
@@ -37,13 +37,23 @@ Details = React.createClass(
     online = <InlineUsers {...@props} users={@props.online} role={2} title='Online Volunteers' />
     campus = <InlineUsers {...@props} users={@props.campus} role={3} title='Campus Volunteers' />
     staff = <InlineUsers {...@props} users={@props.staff} role={4} title='Wiki Ed Staff' />
+    if @props.course.school
+      # FIXME: Convert lego to parameterized messages.
+      school = <p>{CourseUtils.i18n('school', @props.course.string_prefix)}: {@props.course.school}</p>
+    if @props.course.term
+      term = <p>{CourseUtils.i18n('term', @props.course.string_prefix)}: {@props.course.term}</p>
+
+    if @props.course.passcode
+      if @props.current_user.role > 0 || @props.current_user.admin
+        display_code = @props.course.passcode
+      else
+        display_code = '*****'
```

Comment:
@adamwight The passcode is not sent to the client at all unless the user is an admin or instructor in the course. See https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/app/views/courses/course.json.jbuilder#L28 (magic number, whoops)

We could instead just check change the check to `if @props.course.passcode`.